roles/rsyslog/templates/71-additional-log-server.conf.j2

```diff
@@ -1 +1,5 @@
+{% if rsyslog_additional_protocol|default('udp') == 'udp' %}
```

Comment:
Add this var to the role doc?

It is also a bit optimistic to treat everything != udp as tcp, but I'm also not sure it is worth an extra check.